### ISSUE TITLE
A kekulization failure from a SMILES benchmark 

### ISSUE THREAD

**Issue by baoilleach** [rdkit/rdkit#1900]
In results presented at the recent ICCS regarding a SMILES benchmark (https://www.nextmovesoftware.com/products/SMILESBenchmark_ICCS_May2018.pdf), there was a single SMILES string (of the 47K written out by the CDK) that RDKit 2018.03.1 failed to kekulize. Given that (at least) CDK itself, OEChem and Open Babel all succeeded in kekulizing it, it's likely that it is kekulizable.

Unfortunately, it's a buckyball derivative (perhaps the same issue reported by Brian C at https://github.com/rdkit/rdkit/issues/1740?). The CDK aromatic SMILES is:
```
c12c3c4c5c6c7C8c5c-9c%10c%11c%12c%13c%14c%15c%11c9c%16c%17c%15c%18-c%14c%19c%20c%13c(c%12c1c%104)c%21c2c%22c%23c3c6c%24c%23c%25c%26c%22c%21c%20c%27C%19%28C%29(c%18c%30c%17c(C%168)c-%31c%32c%30c%29c(=c%27%26)c%25c%32c%24c7%31)CC=CC%28
```

For reference, the original (Kekule) input to CDK.
```
C12=C3C4=C5C6=C7C8C5=C9C5=C%10C%11=C%12C%13=C%14C%10=C9C9=C%10C%14=C%14C%13=C%13C%15=C%12C(=C%11C1=C54)C1=C2C2=C4C3=C6C3=C4C4=C5C2=C1C%15=C1C%132C6(C%14=C%11C%10=C(C98)C8=C9C%11=C6C(=C15)C4=C9C3=C78)CC=CC2
```

**Comment by greglandrum** [comment]
Confirmed.

**Comment by greglandrum** [comment]
@baoilleach : can you get a kekule SMILES from CDK with the same atom ordering as that given above?

**Comment by baoilleach** [comment]
Will do.

**Comment by bp-kelley** [comment]
This appears to have been resolved at some point (running on 2024.09)

```
>>> m = Chem.MolFromSmiles("C12=C3C4=C5C6=C7C8C5=C9C%10=C%11C%12=C%13C%14=C%15C%11=C9C%16=C%17C%15=C%18C%14=C%19C%20=C%13C(=C%12C1=C%104)C%21=C2C%22=C%23C3=C6C%24=C%23C%25=C%26C%22=C%21C%20=C%27C%19%28C%29(C%18=C%30C%17=C(C%168)C%31=C%32C%30=C%29C(=C%27%26)C%25=C%32C%24=C7%31)CC=CC%28")
>>> Chem.MolToSmiles(m, kekuleSmiles=True)
'C1=CCC23C4=C5C6=C7C8=C9C%10=C%11C%12=C%13C%14=C%15C(=C(C8=C%11%14)C72C1)C1=C3C2=C4C3=C4C7=C8C%11=C%14C%16=C(C%17=C%12C%12C%18=C%19C%20=C%21C(=C6C9=C%20C%10%12)C5=C3C3=C4C4=C(C%19=C%213)C(=C%18%17)C%16=C48)C%13=C%14C%15=C1C%11=C27'
```
